src/pages/Home/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import { useNavigate } from "react-router-dom";
+import styles from "./styles.module.scss";
+
+type Props = {
+    title: string;
+    icon: string;
+    color: string;
+    to: string;
+};
+
+const Card = ({ title, icon, color, to }: Props) => {
+    const navigate = useNavigate();
+    return (
+        <div
+            className={`${styles.card}`}
+            style={{ backgroundColor: color }}
+            onClick={() => navigate(to)}
+        >
+            {/*
```

Comment:
Remove unnecesary comments, if todo still needs to be done then finish it because branch will be merged into dev. Dev branch should be very clean so try to be as clean as possible.